hopsworks-api/src/main/java/io/hops/hopsworks/api/featurestore/FeaturestoreService.java

```diff
@@ -1228,5 +1248,53 @@ public Response updateTrainingDatasetMetadata(@Context SecurityContext sc,
       .entity(trainingDatasetGeneric)
       .build();
   }
+  
+  /**
+   * Verify that the user is allowed to execute the requested operation based on its project role
```

Comment:
@Limmen Typically you refer to a user with `they` or `she`, not `its`

Reply:
Done